src/main/java/org/apache/commons/compress/harmony/unpack200/BandSet.java

```diff
@@ -162,8 +162,8 @@ public long[] parseFlags(final String name, final InputStream in, final int coun
         return parseFlags(name, in, new int[] {count}, (hasHi ? codec : null), codec)[0];
     }
 
-    public long[][] parseFlags(final String name, final InputStream in, final int counts[], final BHSDCodec codec,
-        final boolean hasHi) throws IOException, Pack200Exception {
+    public long[][] parseFlags(final String name, final InputStream in, final int[] counts, final BHSDCodec codec,
+                               final boolean hasHi) throws IOException, Pack200Exception {
```

Comment:
Leave this formatting alone, please; it creates too much noise to review in the PR.

Reply:
Done